src/multisignature.cpp

```diff
@@ -0,0 +1,248 @@
+// Copyright (c) 2017-2018 The PIVX developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <regex>
+#include <boost/algorithm/string/split.hpp>
```

Comment:
Include ordering is non-standard. Should be:
```
in-tree includes

3rd party dependency includes

system includes
```

Also, if we can do this without relying on boost, we should